PyInstaller/hooks/hook-gi.repository.GdkPixbuf.py

```diff
@@ -113,43 +114,42 @@
-                # libdir is actually /lib64. To get around this, we pass the
-                # path to the loader command, and it will create a cache with
-                # the right path.
-                cachedata = exec_command_stdout(gdk_pixbuf_query_loaders,
-                                                *loader_libs)
-
-                cd = []
-                prefix = '"' + os.path.join(libdir, 'gdk-pixbuf-2.0', '2.10.0')
-                plen = len(prefix)
-
-                # For each line in the updated loader cache...
-                for line in cachedata.splitlines():
-                    if line.startswith('#'):
-                        continue
-                    if line.startswith(prefix):
-                        line = '"@executable_path/' + cachedest + line[plen:]
-                    cd.append(line)
-
-                cachedata = u'\n'.join(cd)
-
-                # Write the updated loader cache to this file.
-                with open_file(cachefile, 'w') as fp:
-                    fp.write(cachedata)
-            # Else, GdkPixbuf will do the right thing on Windows, so no changes
-            # to the loader cache are required. For efficiency and reliability,
-            # this command's encoded byte output is written as is without being
-            # decoded.
-            else:
-                with open_file(cachefile, 'wb') as fp:
-                    fp.write(subprocess.check_output(gdk_pixbuf_query_loaders))
+            # To permit string munging, decode the encoded bytes output by
+            # this command (i.e., enable the "universal_newlines" option).
+            #
+            # On Fedora, the default loaders cache is /usr/lib64, but the
+            # libdir is actually /lib64. To get around this, we pass the
+            # path to the loader command, and it will create a cache with
+            # the right path.
+            #
+            # On Windows, the loaders lib directory is relative, starts with
+            # 'lib', and uses \\ as path separators (escaped \).
+            cachedata = exec_command_stdout(gdk_pixbuf_query_loaders,
+                                            *loader_libs)
+
+            cd = []
+            prefix = '"' + os.path.join(libdir, 'gdk-pixbuf-2.0', '2.10.0')
+            plen = len(prefix)
+
+            win_prefix = '"' + '\\\\'.join(['lib', 'gdk-pixbuf-2.0', '2.10.0'])
+            win_plen = len(win_prefix)
+
+            # For each line in the updated loader cache...
+            for line in cachedata.splitlines():
+                if line.startswith('#'):
+                    continue
+                if line.startswith(prefix):
+                    line = '"@executable_path/' + cachedest + line[plen:]
+                elif line.startswith(win_prefix):
+                    line = '"' + cachedest.replace(
+                        '/', '\\\\') + line[win_plen:]
+                cd.append(line)
+
+            cachedata = u'\n'.join(cd)
```

Comment:
```suggestion
            cachedata = '\n'.join(cd)
```

Reply:
Can you also stick `[skip-ci]` in the commit message seeing as our tests for this are ignored. Save AppVeyor 8 hours of going round in circles...

Reply:
Thanks @BoboTiG for the feedback, we definitely don't need to declare strings as unicode any longer 👍 

@bwoodsend I have repushed with [skip-ci]